src/replication.c

```diff
@@ -993,6 +1092,15 @@ void syncCommand(client *c) {
              * resync on purpose when they are not able to partially
              * resync. */
             if (primary_replid[0] != '?') server.stat_sync_partial_err++;
+            if (c->replica_capa & REPLICA_CAPA_RDB_CONN) {
+                c->flags |= CLIENT_REPL_MAIN_CONN;                
+                serverLog(LL_NOTICE,"Replica %s is capable of rdb-connection synchronization, and partial sync isn't possible. "
+                    "Full sync will continue with dedicated RDB connection.", replicationGetReplicaName(c));
+                if (connWrite(c->conn,"+FULLSYNCNEEDED\r\n",17) != 17) {
```

Comment:
`FULLSYNCNEEDED` is pretty much synonyous to `FULLSYNC` IMO. Can we consider the below?
```suggestion
                const char *buf = "+PARALLELFULLSYNC\r\n";
                if (connWrite(c->conn,buf,strlen(buf)) != strlen(buf)) {
```

Reply:
I agree that FULLSYNCNEEDED  is too similar to FULLSYNC , but there is no real parallelize here. 
What do you think of `+DUALCONNECTIONSYNC`?

Reply:
I like `+DUALCONNECTIONSYNC`

Reply:
👍